prime-router/src/main/kotlin/transport/RedoxTransport.kt

```diff
@@ -89,9 +98,19 @@ class RedoxTransport() : ITransport {
             .responseJson()
         // TODO: store the result id when we get line level tracking
         return when (result) {
-            is Result.Success -> true
-            is Result.Failure -> false
+            is Result.Success -> {
+                logger.info("Redox call succeeded: $itemId")
+                true
+            }
+            is Result.Failure -> {
+                logger.error("Redox endpoint call failed: $itemId: ${result.error.message}")
+                false
+            }
             else -> false
         }
     }
+
+    companion object {
+        private val logger: Logger = LogManager.getLogger(RedoxTransport::class.java)
+    }
```

Comment:
Not sure about the propriety of having two different Logger managers in the code, this one, and the context from Azure.  Seems like a recipe for future confusion.  ("I changed my logging level, and I'm still not seeing the  log messages!", etc)

Reply:
@jimduff-usds I followed your lead on logging to resolve the conflict, but by putting Azure Context into the transport interface making transport only work in Azure. I would like to see it to be able to work in other containers as well, like the command line. I don't see any disadvantage of using Log4J, because it works in all containers.